Lib/threading.py

```diff
@@ -1109,7 +1109,7 @@ def is_alive(self):
 
         This method returns True just before the run() method starts until just
         after the run() method terminates. The module function enumerate()
-        returns a list of all alive threads.
+        returns a list that contains all alive threads.
```

Comment:
I think the existing wording is fine here; not sure why this change is need?

Reply:
That's because it may also contain a terminated main thread.

Reply:
That makes the new wording correct, but possibly misleading.

The sentence about `enumerate()` seems somewhat out of place here; perhaps it could just be removed? Otherwise let's add something like ("and always the main thread") at the end.

Reply:
Perhaps just "see also .."?